hydra_python_core/doc_writer.py

```diff
@@ -673,8 +673,7 @@ def __init__(self, code: int, id_: str = None, title: str = "", desc: str = "")
     def generate(self, status_type: str = "Status") -> Dict[str, Any]:
         """Get as Python dict."""
         status = {
-            "@context": ("https://raw.githubusercontent.com/HydraCG/Specifications"
-                         "/master/spec/latest/core/core.jsonld"),
+            "@context": ("https://www.w3.org/ns/hydra/core"),
```

Comment:
the parenthesis were present because it was a multi-line string. Now that it is a single-line the parenthesis are useless